src/zb_links/wsgi.py

```diff
@@ -1,3 +1,17 @@
-from zb_links.app import create_app
+from zb_links import config
+from zb_links.app import create_app as create_api_app
+from zb_links.db.init_tables.init_db_app import create_app as create_db_app
 
-application = create_app()
+
+application = None
+try:
+    flask_app = config["application"]["FLASK_APP"]
+
+    if flask_app == "zb_links.app":
+        application = create_api_app()
+    if flask_app == "zb_links.db.init_tables.init_db_app.py":
+        application = create_db_app()
```

Comment:
How about:
```python
application = create_db_app() if flask_app == "zb_links.db.init_tables.init_db_app.py"  else create_api_app()
```

Reply:
using the ideas concerning the default app above the following could be used:

application = None
try:
    flask_app = config["application"]["DEFAULT_APP"]
    try:
        flask_app = config["application"]["CUSTOM_APP"]
    except KeyError:
        pass

    if flask_app == "zb_links.db.init_tables.init_db_app.py":
        application = create_db_app()
    else:
        application = create_api_app()
except KeyError:
    application = create_api_app()

i don't know if this is any better than what is already there, although i like the idea of removing the if flask_app=='zb_links.app' line.  The use of the DEFAULT_APP parameter makes it easy to read as well.

Reply:
but then a danger arises if DEFAULT_APP is commented out in the config file. 

i would just leave it as is.

Reply:
I think for this code, it is not that relevant. In the particular case, one might consider merging create_db_app and create_api_app and resolve the issue inside that function. However, in general, I think we should strive for readable code and reduce complexity whenever possible.